runtime/doc/news.txt

```diff
@@ -113,6 +113,7 @@ LSP
 
 • Completion side effects (including snippet expansion, execution of commands
   and application of additional text edits) is now built-in.
+• |vim.lsp.util.locations_to_items()| correctly sets `end_col` field.
```

Comment:
I wouldn't consider this a new feature (it almost feels like a bug fix), but I'm unsure.

Reply:
I kinda disagree because `end_col` in qf items is relatively new.

Reply:
Sorry, the "correctly" in the news threw me.